### ISSUE TITLE
Error on https://game192277.konggames.com/games/Kongregate/sandbox-hero/frame/c3e9b630-8249-4497-ba58-80b2c96b9490/

### ISSUE THREAD

**Issue by deividbinev12** [ruffle-rs/ruffle#9848]
# Error Info
Error name: Error
Error message: panicked at 'wgpu error: Validation Error

Caused by:
    In CommandEncoder::copy_buffer_to_texture
    Copy error
    bytes per row does not respect `COPY_BYTES_PER_ROW_ALIGNMENT`

', /home/runner/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/c585127/wgpu/src/backend/direct.rs:2973:5
Error stack:
```
Error: panicked at 'wgpu error: Validation Error

Caused by:
    In CommandEncoder::copy_buffer_to_texture
    Copy error
    bytes per row does not respect `COPY_BYTES_PER_ROW_ALIGNMENT`

', /home/runner/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/c585127/wgpu/src/backend/direct.rs:2973:5
    at W.n.wbg.__wbg_new_15d3966e9981a196 (https://unpkg.com/@ruffle-rs/ruffle/core.ruffle.e2e407f6cdebdbdcde57.js:1:46442)
    at std::sync::once::Once::call_once::{{closure}}::h97aad3534cecf95d (wasm://wasm/0293e1ca:wasm-function[2038]:0x4e93a9)
    at std::sys_common::once::generic::Once::call::h6b012ac601cc39bf (wasm://wasm/0293e1ca:wasm-function[1973]:0x4db496)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h58f54bbbf27f6496 (wasm://wasm/0293e1ca:wasm-function[7457]:0x7b20f6)
    at std::panicking::rust_panic_with_hook::hbafe3e603d331223 (wasm://wasm/0293e1ca:wasm-function[5249]:0x70a1d3)
    at core::panicking::panic_fmt::h1d17fc068f528130 (wasm://wasm/0293e1ca:wasm-function[7786]:0x7c1398)
    at core::ops::function::Fn::call::h323155b933db1a36 (wasm://wasm/0293e1ca:wasm-function[5493]:0x71b44b)
    at wgpu::backend::direct::ErrorSinkRaw::handle_error::hd79246f92cb7548d (wasm://wasm/0293e1ca:wasm-function[4105]:0x6a2fc0)
    at wgpu::backend::direct::Context::handle_error_nolabel::h8181bf803debd3db (wasm://wasm/0293e1ca:wasm-function[2388]:0x56440e)
    at <T as wgpu::context::DynContext>::command_encoder_copy_buffer_to_texture::hfb5338c8e9227e2e (wasm://wasm/0293e1ca:wasm-function[5580]:0x720943)
```
AVM2 stack:
```
    at flash.display3D::Context3D/flash::display3D::Context3D::present()
    at starling.core::Starling/render()
    at starling.core::Starling/nextFrame()
    at starling.core::Starling/onEnterFrame()
```

# Player Info
Allows script access: true

# Page Info
Page URL: https://game192277.konggames.com/games/Kongregate/sandbox-hero/frame/c3e9b630-8249-4497-ba58-80b2c96b9490/?kongregate_host=www.kongregate.com
SWF URL: https://game192277.konggames.com/gamez/0019/2277/live/SandboxHero.swf?kongregate_game_version=1439420770

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36 Edg/110.0.1587.63
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-03-04
Channel: nightly
Built: 2023-03-04T00:19:50.818Z
Commit: 1ef539e31bc58569d10abda9a70ab9c0d5c03349
Is extension: false

# Metadata
width: 1000
height: 600
frameRate: 60
numFrames: 3
swfVersion: 20
backgroundColor: #000000
isActionScript3: true
uncompressedLength: 20748933

**Comment by n0samu** [comment]
This was probably fixed by #14345, but by then the game was already regressed - #14596

**Comment by n0samu** [comment]
Indeed, fixed by #14345
